### PR TITLE
Wai Laam "Josie" Lui - HW01

### PR DESCRIPTION
**Pull request by luijosie**
Reflection
I have used RMarkdown for quite a while, so the markdown task was relatively swift for me. I had to google around about Git setup for Windows, and I ended up with a bash/SSH setup with a bit of luck. Tracking which files to commit was quite new to me. Overall version control is very helpful.

### CODE REVIEW

**Review by leahvf — COMMENTED** [review]
Evaluation
----------------------------------------------------
| Topic                       | Excellent | Satisfactory | Needs Work |
|-----------------------------|-----------|--------------|------------|
| **Coding style**            |     X      |              |            |
| **Coding strategy**         |     X      |              |            |
| **Presentation**            |           |       X       |            |
| **Achievement, creativity** |     X      |              |            |
| **Ease of access**          |      X     |              |            |

Remarks:
I really enjoyed reading your introduction and feel like I learned a lot about you! Your food Instagram is awesome :star:
I learned what the term **SSH** means. I thought that the lists with subcategories were a nice touch and helped to keep your page organized. The one thing I would have done differently is to add a picture.

**Review by deblnia — COMMENTED** [review]
Evaluation
----------------------------------------------------
| Topic                       | Excellent | Satisfactory | Needs Work |
|-----------------------------|-----------|--------------|------------|
| **Coding style**            |   X       |              |            |
| **Coding strategy**         |    X      |              |            |
| **Presentation**            |    X       |              |            |
| **Achievement, creativity** |           |    X          |            |
| **Ease of access**          |           |        X      |            |

Remarks:

* Successfully used 4 elements of markdown syntax. 
* Good commit history and GitHub reflection. 
* Try and play around with other elements of markdown syntax (i.e. images).

File: README.md [hunk 1]
```diff
@@ -1,2 +1,43 @@
...
+
+Josie Lui's repository for homework 01
+
+### Wai Laam (Josie) Lui
+
+Created on: 4/7/2019 
+
+### Who Am I? 
+
+Hi there! I'm **Josie**, a second-year undergrad at UChicago pursuing degrees in Political Science and Statistics. I'm originally from Hong Kong but went to high school in Los Angeles and Bellevue, Washington.
```
Comment (hannahem) [inline README.md:11]: Hey Josie! I grew up in Issaquah, right next to Bellevue!

File: README.md [hunk 2]
```diff
@@ -1,2 +1,43 @@
...
+
+### What I Like to Do? 
+
+* Skiing 
+	* The big mountains give me inspiration and remind me of my limitations 
+* Yoga 
+	* I *LOVE* Bikram Yoga
+* Cooking/Cuisine 
+	* One of the item on my to-do list is to pursue a diploma at Le Cordon Bleu! 
+	* My foodiegram [@jothefoodie](https://www.instagram.com/jothefoodie_/).
```
Comment (hannahem) [inline README.md:21]: Great use of a link!

**Review by hannahem — COMMENTED** [review]
Evaluation
----------------------------------------------------
| Topic                       | Excellent | Satisfactory | Needs Work |
|-----------------------------|-----------|--------------|------------|
| **Coding style**            |    X       |              |            |
| **Coding strategy**         |     X      |              |            |
| **Presentation**            |     X      |              |            |
| **Achievement, creativity** |     X      |              |            |
| **Ease of access**          |     X      |              |            |

Remarks:
Hi Josie, your bio was fun, creative, and taught me so much about you! It also looks great -- I did not see any errors in your code. Your use of lists and sublists, bolding, italicization, and headers made it easy and fun to read your bio -- it had a great flow to it. I especially liked that you supported your Cooking/Cuisine bullet point with a link to your foodiegram -- can't wait to check it out! One small improvement would be to add an "s" to the end of "cat" on the "Persian cat" mention. Overall, awesome job and I'm eager to see more of your work!

**Pull request closed without merging**